source/api/search/0.9/index.md

```diff
@@ -471,11 +471,10 @@ The result might be:
 
 #### 3.4.4. Multi-Annotations Hits
 
-There may be multiple annotations that match a single phrase search.  If the following line of the text was a continuation of the sentence "and they are worth one in the hand", and the user searched for the phrase "bush and", the match would span multiple annotations regardless of whether the annotation was at word or line level.
+Depending on the way that the text is made available via annotations, such as word by word or line by line, there may be multiple annotations that match a single search.  For example, if the first line was "Two birds in the bush", the second line was "are worth one in the hand", and the user search for the phrase "bush are", the match would span both of the line based annotations.  If the annotations are at word level, then phrases will require multiple annotations.
```

Comment:
I think "Depending on the way that the text is made available via annotations" needs to be expanded. We know it means you may have Alto or hOCR or native IIIF or some other format driving your service, but I'm not sure that this would be clear on a first reading.

Also mayb s/user search/client search/ or else go back to "user search_ed_"